File: 프로그래머스/1/161990. 바탕화면 정리/바탕화면 정리.py
def solution(wallpaper):
    answer = [len(wallpaper), len(wallpaper[0]), 0, 0]
    for idx, str in enumerate(wallpaper):
        if "#" in str:
            answer[0] = min(idx, answer[0])
            answer[2] = max(idx+1, answer[2])
            answer[1] = min(str.find("#"), answer[1])
            answer[3] = max(str.rfind("#")+1, answer[3])
    return answer

File: 프로그래머스/1/161990. 바탕화면 정리/바탕화면 정리.java
import java.util.*;
class Solution {
    public int[] solution(String[] wallpaper) {
        int[] answer = new int[4];
        int minX = Integer.MAX_VALUE;
        int minY = Integer.MAX_VALUE;
        int maxX = Integer.MIN_VALUE;
        int maxY = Integer.MIN_VALUE;
        for (int i = 0; i < wallpaper.length; i++) {
            for (int j = 0; j < wallpaper[i].length(); j++) {
                if (wallpaper[i].charAt(j) == '#') {
                    minX = Integer.min(minX, i);
                    minY = Integer.min(minY, j);
                    maxX = Integer.max(maxX, i);
                    maxY = Integer.max(maxY, j);
                }
            }
        }
        answer[0] = minX;
        answer[1] = minY;
        answer[2] = maxX+1;
        answer[3] = maxY+1;
        return answer;
    }
}